extensions/2.0/Khronos/KHR_displaymapping_pq/README.md

```diff
@@ -0,0 +1,398 @@
+
+This section describes how light contribution values shall be handled.  
+
+When the KHR_displaymapping_pq extension is used all lighting and pixel calculations shall be done using the value 10000 (cd / m2) as the maximum ouput brightness.  
+This does not have an impact on color texture sources since they define values as contribution factor.  
+The value 10000 cd / m2 for an output pixel with full brightness is chosen to be compatible with the Perceptual Quantizer (PQ) used in the SMPTE ST 2084 transfer function.  
+
+When using this extension light contribution values shall be aligned to account for 10000 cd/m2 as max luminance, meaning that the dynamic range is 0 to 10000.    
+This means that content creators shall be aware of 10000 cd/m2 as the maximum brightness value range.  
+It does not mean that the display will be capable of outputing at this light luminance.  
+
+## Exporter considerations for light contribution values
+
+This section describes how an exporter shal handle lightsources, such as point, directional or environment lights, that are saved with the model.  
+
+A content creation tool supporting this extension shall sum upp light contribution for a scene before exporting to glTF, this can be a naive addition of all lights included in the scene that adds max values together.  
+If scene max light contribution is above 10000 cd / m2 there is a choice to either downscale light values before export or to set `sceneAperture` to `AUTO`  
+This means that implementations will calculate max light intensity for the scene and use as a scale factor to keep all light contribution within 0 - 10000 cd / m2.  
+
+Light contribution values above 10000 cd / m2 is strongly discouraged and will be clamped by implementations if not adjusted by scene aperture.    
+
+## Displaymapping
+
+This section describes how the mapping of internal, scene linear light, values to display output values in the range [0 - 1.0] shall be done.  
+
+Displaymapping will be done different depending on wether the output is an HDR or SDR capable display.  
+If the output type is unknown then it shall be considered to be SDR, and the SDR ootf shall be used.  
+
+### To HDR capable display
+
+To convert from internal values (linear scene light) to the non-linear output value in the range 0.0 - 1.0 the reference PQ OETF shall be used. [See OETF](#oetf)  
+This is specified in ITU BT.2100:
+https://www.itu.int/rec/R-REC-BT.2100/en  
+
+If the framebuffer format and colorspace is known to the implementation then a format and colorspace shall be chosen to preserve the range and precision of the SMPTE ST 2084 transfer function.  
+If available, a framebuffer colorspace that is compatible with the color primaries of ITU BT.2020 shall be used.  
+If this colorspace is used the source color images must be converted from linear ITU BT.709 values to ITU BT.2020 linear values, where applicable depending on texture target and source format.  
```

Comment:
Note there is another solution: for a real-time renderer, there are not many benefits to rendering directly in the Rec.2020 gamut (path tracers do get some benefits in bounce lighting). You can instead render in Rec.709 and convert to Rec.2020 as a final pass before any tone mapping, transfer function, etc. This can be much easier/cheaper than having to convert every input.

Reply:
Agreed. This process should not be normatively defined here.

Reply:
Updated according to comments.
Implementations are free to do conversion at any stage before value is written to framebuffer.